packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -1973,6 +1973,27 @@ export class EntityService {
           );
         }
 
+        const resourceWithProject = await this.prisma.resource.findUnique({
+          where: {
+            id: entity.resourceId,
+          },
+          include: {
+            project: true,
+          },
+        });
```

Comment:
same comment as for the previous PR related to `resourceWithProject` 🙃

Reply:
same comment as for the previous PR related to resourceWithProject 🙃